### ISSUE TITLE
new_feature

### ISSUE THREAD

**Issue by AuraMoon553** [Harsha200105/DesktopAssitant#14]
#this feature can be useless or may be a fun #feature thats upto usage and user but i thought #it would have been great if this was a part

elif 'tts' in query :
           query = query.replace("tts", "")
           results = (query)
          speak(results)

**Comment by Harsha200105** [comment]
I don't think this feature is required for the jarvis to operate @AuraMoon553 !!!
